safe/impact_functions/base.py

```diff
@@ -107,6 +102,12 @@
 LOGO_ELEMENT = m.Brand()
 LOGGER = logging.getLogger('InaSAFE')
 
+__author__ = 'akbargumbira@gmail.com'
```

Comment:
Do we now put the "docs" after the import? If so, it should be above the LOGO_ELEMENT in line 102 and we need to do for all files :)